src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,23 @@
+.calendar {
+  display: flex;
+  gap: $gap;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  padding: 20px $padding-calendar;
+
+  &__day {
+    @include square ($size-day-box);
+    background-color: #eee;
+    border: 1px solid #000;
+    text-align: center;
+    line-height: $size-day-box;
+
+    &:hover {
+      background: #faafd3;
+      cursor: pointer;
+      transition: background-color 0.5s, transform 0.5s;
+      transform: matrix3d(1, 0, 1, 0, 0, 1, 1, 0, 0, 0, 1, (1/1000), 0, 0, 0, 1)
```

Comment:
Be attentive to details, it will help you on your future job

![image](https://user-images.githubusercontent.com/57747481/174023520-576caec9-02c5-42f1-ba37-2f19f69dd7b5.png)